dags/atd_trail_counters.py

```diff
@@ -55,7 +55,7 @@
 ) as dag:
     env_vars = get_env_vars_task(REQUIRED_SECRETS)
 
-    start = "{{ prev_start_date_success.strftime('%Y-%m-%d') if prev_start_date_success else '1970-01-01'}}"
+    start = "{{ (prev_start_date_success - macros.timedelta(days=7)).strftime('%Y-%m-%d') if prev_start_date_success else '1970-01-01'}}"
```

Comment:
`macros` comes from [this thread](https://stackoverflow.com/questions/54517369/airflow-use-execution-date-datetime-is-undefined) on stack overflow